test/build-utils.ts

```diff
@@ -0,0 +1,97 @@
+/**
```

Comment:
This file is used by test code only. The contents were mostly moved here from `build.spec.ts`, so that it could be reused / shared with the new `multibuild.spec.ts`.